mod/profile.php

```diff
@@ -6,78 +6,81 @@
 use Friendica\Core\System;
 use Friendica\Database\DBM;
 
-require_once('include/contact_widgets.php');
-require_once('include/redir.php');
+require_once 'include/contact_widgets.php';
+require_once 'include/redir.php';
 
-function profile_init(App $a) {
-
-	if(! x($a->page,'aside'))
+function profile_init(App $a)
+{
+	if (!x($a->page, 'aside')) {
 		$a->page['aside'] = '';
+	}
 
-	if($a->argc > 1)
+	if ($a->argc > 1) {
 		$which = htmlspecialchars($a->argv[1]);
-	else {
-		$r = q("select nickname from user where blocked = 0 and account_expired = 0 and account_removed = 0 and verified = 1 order by rand() limit 1");
+	} else {
+		$r = q("SELECT `nickname` FROM `user` WHERE `blocked` = 0 AND `account_expired` = 0 AND `account_removed` = 0 AND `verified` = 1 ORDER BY RAND() LIMIT 1");
 		if (DBM::is_result($r)) {
 			goaway(System::baseUrl() . '/profile/' . $r[0]['nickname']);
-		}
-		else {
+		} else {
 			logger('profile error: mod_profile ' . $a->query_string, LOGGER_DEBUG);
-			notice( t('Requested profile is not available.') . EOL );
+			notice(t('Requested profile is not available.') . EOL);
 			$a->error = 404;
 			return;
 		}
 	}
 
 	$profile = 0;
-	if((local_user()) && ($a->argc > 2) && ($a->argv[2] === 'view')) {
+	if (local_user() && $a->argc > 2 && $a->argv[2] === 'view') {
 		$which = $a->user['nickname'];
 		$profile = htmlspecialchars($a->argv[1]);
-	}
-	else {
+	} else {
 		auto_redir($a, $which);
 	}
 
-	profile_load($a,$which,$profile);
+	profile_load($a, $which, $profile);
 
-	$blocked = (((Config::get('system','block_public')) && (! local_user()) && (! remote_user())) ? true : false);
-	$userblock = (($a->profile['hidewall'] && (! local_user()) && (! remote_user())) ? true : false);
+	$blocked = Config::get('system', 'block_public') && !local_user() && !remote_user();
```

Comment:
Thanks for improving this. The previous line generated some "WTF" from my side. I just did a short check: We have several of these places in the code.

Reply:
I know, but it's one of those tangents I know I wouldn't get out of easily, you better check out my latest monster PR.